monitoring/monitoring-client.py

```diff
@@ -22,17 +22,22 @@
 
 def alert(data):
     print_to_file(data)
-    url = config['DEFAULT']['slack']
-    headers = {'Content-type': 'application/json'}
-    try:
-        r = requests.post(
-            url,
-            data=json.dumps({'text': '{}'.format(data)}),
-            headers=headers,
-        )
-    except Exception as e: 
-        print_to_file("Unable to send alert - {}".format(e))
-    print_to_file("Alerting - {}: {}".format(r.status_code, r.reason))
+    if monitor_config['DEFAULT']['discord']:
+        url = monitor_config['DEFAULT']['discord']
+        info = {'username': 'piman', 'content': 'Alert:', 'embeds': []}
```

Comment:
Are you talking about 'username': 'piman'? Because that just refers to the username that is shown as the post creator in discord, it could be anything (even empty I think). I could change that if you want. Besides that, I don't think I use userids or passwords anywhere in my change, can you clarify?

Reply:
ok that's ok.